### PR TITLE
#125 hack aubg animation

### PR DESCRIPTION
**Pull request by SuelaCark**
Added initial structure, gif background image and fixed responsiveness

### CODE REVIEW

File: packages/web/src/components/spa/HackAUBG/LandingAnimation/InvokeAnimation.jsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { AnimateMatrix } from './landing_animate';
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/LandingAnimation/InvokeAnimation.jsx:1]: This file should be removed

File: packages/web/src/components/spa/HackAUBG/LandingAnimation/LandingAnimation.jsx [hunk 2]
```diff
@@ -1,21 +1,26 @@
 import React from 'react';
+import './landing_animation.css';
+// import { InvokeAnimation } from './InvokeAnimation';
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/LandingAnimation/LandingAnimation.jsx:3]: Please remove comments which are not describing any specific behaviour

File: packages/web/src/components/spa/HackAUBG/LandingAnimation/landing_animate.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+// Initialising the canvas
+export const AnimateMatrix = () => {
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/LandingAnimation/landing_animate.js:2]: This file should be removed as well.

File: packages/web/src/components/spa/HackAUBG/LandingAnimation/landing_animation.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    height: 30vw;
+    background: #000;
+    /* background-image: url(https://j.gifs.com/Q1xW4q@large.gif?download=true); */
+    /* background-image: url(https://img.gs/czjpqfbdkz/full/https://raw.githubusercontent.com/mov-rax-rbx/rsmatrix/3a5a42c5dd585c707637d9f62f7c93f95405d7f7/gif/preview2.gif); */
+    background-image: url('/public/gif.gif');
+    background-size: cover;
+    background: fit-content;
+    align-items: center;
+    justify-content: center;
+    display: flex;
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/LandingAnimation/landing_animation.css:15]: Remove leftover css for the js animation if any.

**Review by asynchroza — CHANGES_REQUESTED** [review]

**Comment by asynchroza** [comment]
Delete redundant gif file as well.

**Comment by SuelaCark** [comment]
Please dont merge this PR yet, there are some responsiveness issues that I have to resolve first.

**Comment by asynchroza** [comment]
Remove old animation files

**Comment by asynchroza** [comment]
@SuelaCark click on resolve whenever you're done applying the specific changes. Most are resolved now but there is still a leftover file which shouldn't be there.

Remove it and ping somebody on @AUBGTheHUB/nosyncdev to approve it tomorrow.

**Comment by SuelaCark** [comment]
@AUBGTheHUB/nosyncdev Please review this PR. (@r-mishev @Aleks1908 )

**Review by Aleks1908 — APPROVED** [review]
I think everything looks nice on both desktop and mobile

**Pull request merged**